grafana/config.js.tmpl

```diff
@@ -0,0 +1,27 @@
+define(['settings'],
+function (Settings) {
+  return new Settings({
+    datasources: {
+      'metrics': {
+        type: 'influxdb',
+        url: "/db/{{ .Env.INFLUXDB_NAME }}",
```

Comment:
The 'url' needs to be that of where Grafana is accessible (ex. :`https://<master>/api/v1beta1/proxy/services/monitoring-grafana`). 
The intention is to make Grafana client side re-use its URL for accessing influxdb.